src/CitySearch.js

```diff
@@ -1,16 +1,17 @@
 import React, { Component } from 'react';
-import NumberOfEvents from './NumberOfEvents';
+import PropTypes from 'react';
```

Comment:
This should be `import PropTypes from 'prop-types';` otherwise the app won't start. Please fix it before merging the pull request.